arangod/RocksDBEngine/RocksDBMetadata.cpp

```diff
@@ -723,3 +760,15 @@ void RocksDBMetadata::loadInitialNumberDocuments() {
   }
   return Result();
 }
+
+/*static*/ Result RocksDBMetadata::deleteIndexEstimateBatch(rocksdb::DB* db, uint64_t objectId, rocksdb::WriteBatch& wb) {
```

Comment:
`db` does not seem to be used here. Shall we remove it from the signature?
```suggestion
/*static*/ Result RocksDBMetadata::deleteIndexEstimateBatch(uint64_t objectId, rocksdb::WriteBatch& wb) {
```